website/content/api-docs/secret/databases/oracle.mdx

```diff
@@ -130,3 +130,10 @@ list the plugin does not support that statement type.
   base64-encoded semicolon-separated string, a serialized JSON string array, or
   a base64-encoded serialized JSON string array. The `{{name}}` value will be
   substituted. If not provided defaults to a generic drop user statement.
+
+- `rotation_statements` `(list: [])` – Specifies the database statements to be
+  executed to rotate the password for a given username. Must be a
+  semicolon-separated string, a base64-encoded semicolon-separated string, a
+  serialized JSON string array, or a base64-encoded serialized JSON string
+  array. The `{{name}}` and `{{password}}` values will be substituted. The
```

Comment:
Does the substitution happen in cases?

Reply:
What do you mean by cases?

Reply:
Ah, I think I misunderstood. The substitution is happening on the decoded config.